doc/indexing.rst

```diff
@@ -36,6 +36,8 @@ below and summarized in this table:
 | By name          | By label     | ``arr.sel(space='IA')`` or |br| | ``ds.sel(space='IA')`` or |br| |
 |                  |              | ``arr.loc[dict(space='IA')]``   | ``ds.loc[dict(space='IA')]``   |
 +------------------+--------------+---------------------------------+--------------------------------+
+| By name          | By integers  | ``arr.isel_points(x=[0, 1])``   | ``ds.isel_points(x=[0, 1])``   |
```

Comment:
I would leave this outside the table for now -- it doesn't quite fit in with the other methods, because it does pointwise indexing, not outer/orthogonal indexing.